cmd/minishift/cmd/start.go

```diff
@@ -419,6 +425,11 @@ func initClusterUpFlags() *flag.FlagSet {
 	clusterUpFlagSet.AddFlag(cmdutil.HttpProxyFlag)
 	clusterUpFlagSet.AddFlag(cmdutil.HttpsProxyFlag)
 
+	if future {
```

Comment:
hiding feels like it is not helpful to a user to see what we support/offer as features.

using `ENABLE_EXPERIMENTAL` is a choice already, so assist them as much as possible... since they already made a decision to enable a feature that might fail.